scalding-core/src/main/scala/com/twitter/scalding/filecache/DistributedCacheFile.scala

```diff
@@ -105,16 +105,14 @@ final case class UncachedFile private[scalding] (source: Either[String, URI]) {
   }
 
   private[this] def addHdfs(conf: Configuration): CachedFile = {
-    HDistributedCache.createSymlink(conf)
```

Comment:
Because the original call to the method doesn't do anything.

```
  /**
   * Originally intended to enable symlinks, but currently symlinks cannot be
   * disabled. This is a NO-OP.
   * @param conf the jobconf
   * @deprecated This is a NO-OP. 
   */
  @Deprecated
  public static void createSymlink(Configuration conf){
    //NOOP
  }
```